deltachat-ios/View/ChatTitleView.swift

```diff
@@ -3,20 +3,41 @@ import DcCore
 
 class ChatTitleView: UIView {
 
-    private var titleLabel: UILabel = {
+    private lazy var titleLabel: UILabel = {
         let titleLabel = UILabel()
         titleLabel.translatesAutoresizingMaskIntoConstraints = false
         titleLabel.backgroundColor = UIColor.clear
         titleLabel.font = UIFont.systemFont(ofSize: 16, weight: .semibold)
         titleLabel.textAlignment = .center
+        titleLabel.setContentHuggingPriority(.defaultHigh, for: .horizontal)
         return titleLabel
     }()
 
-    private var subtitleLabel: UILabel = {
+    private lazy var verifiedView: UIImageView = {
+        let imgView = UIImageView()
+        let img = UIImage(named: "verified")?.scaleDownImage(toMax: 18)
```

Comment:
this seems a bit too large, maybe we  can also use sth. as `titleLabel.font.pointSize * 0.9` here - that way it would also scale together with font size.

Reply:
Font sizes for the title bar are hard coded since UIController titles usually change their font size only minimally (mostly capable to reduce the seize, but not increase). I reduce the hard-coded value to 0.9 x 16 points (font size of title text size).